packages/notifi-core/lib/NotifiClient.ts

```diff
@@ -29,11 +29,21 @@ export type AlertFrequency =
   | 'HOURLY'
   | 'DAILY';
 
+export type ValueItemConfig = Readonly<{
+  key: string;
+  op: 'lt' | 'lte' | 'eq' | 'gt' | 'gte';
+  value: string;
```

Comment:
value should be a number

Reply:
i was thinking we'd serialize it to string and let the backend do the explicit conversions to the relevant values to match (based on key, i guess?) -- wdyt? this would let us use things like `equality` on strings

an alternative might be to encode expected type in the API
```
export type ValueItemConfig = Readonly<
  | { key: string, type: 'number', op: NumberOp, value: number }
  | { key: string, type: 'string', op: StringOp, value: string }
    ...
>;
```